### PR TITLE
Fixed typo

### PR DESCRIPTION
**Pull request by brandonferens**
A good bourbon needs to be both aged and spelled properly.

### CODE REVIEW

File: content/collections/docs/taxonomies.md [hunk 1]
```diff
@@ -225,7 +225,7 @@ In the YAML file, add data like so:
 
 ``` .language-yaml
 food: bacon
-drink: whisky
+drink: whiskey
 ```
 
 Whenever referencing the Terms in your templates, now `{{ food }}` and `{{ drink }}` would output `bacon` and `whiskey` respectively.
```
Comment (jasonvarga) [inline content/collections/docs/taxonomies.md]: Can you remove the `e` from `whiskey` here instead? We are scotch drinkers! 🥃

**Review by jasonvarga — CHANGES_REQUESTED** [review]

File: content/collections/docs/taxonomies.md (hunk 1 above)
Comment (brandonferens) [reply content/collections/docs/taxonomies.md]: Well snap.

Comment (jasonvarga) [reply content/collections/docs/taxonomies.md]: (If that's a pain just say so and I'll tweak it)

Comment (brandonferens) [reply content/collections/docs/taxonomies.md]: Updated. Highland or lowland scotch. Which do you prefer?

Comment (jasonvarga) [reply content/collections/docs/taxonomies.md]: Islay actually if that's an option. A good bourbon doesn't hurt either though.

**Comment by jasonvarga** [comment]
Thanks! 😊 🥃

**Pull request merged**